great_expectations/datasource/batch_kwargs_generator/batch_kwargs_generator.py

```diff
@@ -210,9 +210,10 @@ def reset_iterator(self, generator_asset=None, data_asset_name=None, **kwargs):
             not generator_asset and data_asset_name
         ), "Please provide either generator_asset or data_asset_name."
         if generator_asset:
+            # deprecated-v0.11.0
             warnings.warn(
-                "The 'generator_asset' argument will be deprecated and renamed to 'data_asset_name'. "
-                "Please update code accordingly.",
+                "The 'generator_asset' argument is deprecated as of v0.11.0 and will be removed in v0.16. "
+                "Please use 'data_asset_name' moving forward.",
```

Comment:
I think this "moving forward" language should be standardized to "instead" (as in other warnings)

Reply:
done! changed throughout the codebase in the next commit.